src/application/index.ts

```diff
@@ -202,6 +202,51 @@ function configureEsLint(project: ProjectDefinition) {
 function addDependenciesToPackageJson() {
   return (host: Tree) => {
     [
+      {
+        type: NodeDependencyType.Default,
+        name: '@angular/flex-layout',
+        version: '12.0.0-beta.34'
+      },
+      {
+        type: NodeDependencyType.Default,
+        name: '@angular/material',
```

Comment:
Angular material wordt niet altijd gebruikt in projecten he. Voor Constructiv wordt bv ng-bootstrap gebruikt ipv angular material

Reply:
Klopt, maar in onze vernaculair hadden we wel Angular Material afgeklopt dus ik vind het wel logisch dat we deze hier mee in verwerken.

Reply:
Oké!